SingularityUI/app/handlebarsHelpers.coffee

```diff
@@ -21,6 +24,9 @@ Handlebars.registerHelper "ifAll", (conditions..., options)->
         return options.inverse @ unless condition?
     options.fn @
 
+Handlebars.registerHelper 'ifNotEmptyArray', (arr, options) ->
```

Comment:
Nope! I originally was using it but forgot to delete it when I realized that it's not at all necessary to have a function like that.